d2l-button-toggle-group.js

```diff
@@ -0,0 +1,156 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import './d2l-button-toggle.js';
+
+/**
+ * @customElement
+ * @polymer
+ */
+export class D2LButtonToggleGroup extends PolymerElement {
+	static get is() { return 'd2l-button-toggle-group'; }
+	static get template() {
+		return html`
+			<style>
+				div ::slotted(.d2l-button-toggle-grouped) {
+					--d2l-button-toggle-border-radius: 0;
+					--d2l-button-toggle-border-width: 1px 1px 1px 0;
+				}
+				:host([orientation="vertical"]) div ::slotted(.d2l-button-toggle-grouped) {
+					--d2l-button-toggle-border-width: 0 1px 1px 1px;
+				}
+				div ::slotted(.d2l-button-toggle-grouped:first-child),
+				:host([dir="rtl"]) div ::slotted(.d2l-button-toggle-grouped:last-child) {
+					--d2l-button-toggle-border-radius: 0.3rem 0 0 0.3rem;
+					--d2l-button-toggle-border-width: 1px;
+				}
+				div ::slotted(.d2l-button-toggle-grouped:last-child),
+				:host([dir="rtl"]) div ::slotted(.d2l-button-toggle-grouped:first-child) {
+					--d2l-button-toggle-border-radius: 0 0.3rem 0.3rem 0;
+				}
+				:host([orientation="vertical"]) div ::slotted(.d2l-button-toggle-grouped:first-child) {
+					--d2l-button-toggle-border-radius: 0.3rem 0.3rem 0 0;
+					--d2l-button-toggle-border-width: 1px;
+				}
+				:host([orientation="vertical"]) div ::slotted(.d2l-button-toggle-grouped:last-child) {
+					--d2l-button-toggle-border-radius: 0 0 0.3rem 0.3rem;
+				}
+				div {
+					display: inline-flex;
+					flex-wrap: nowrap;
+					outline: none;
+				}
+				:host([orientation="vertical"]) div {
+					flex-direction: column;
+				}
+			</style>
+			<div aria-role="toolbar" aria-label="[[label]]">
+				<slot></slot>
+			</div>
+		`;
+	}
+	static get properties() {
+		return {
+			disabled: {
+				type: Boolean,
+				value: false,
+				observer: '_disabledChanged'
+			},
+			label: {
+				type: String,
+				value: false
+			},
+			orientation: {
+				type: String,
+				value: 'horizontal'
+			},
+			_lastFocused: {
+				type: Object,
+				value: {}
+			},
+			_buttons: {
+				type: Array,
+				value: []
+			}
+		};
+	}
+	static get observers() {
+		return [
+			'_disabledChanged(disabled, _buttons)'
+		];
+	}
+
+	ready() {
+		super.ready();
+		this._initializeButtons();
+
+		this.addEventListener('keydown', this._handleKeyDown);
+		this.addEventListener('click', this._handleClick);
+	}
+
+	_disabledChanged() {
+		this._buttons.forEach(b => b.disabled = this.disabled);
+	}
+
+	_getButtons() {
+		return this.querySelectorAll('*');
+	}
+
+	_initializeButtons() {
+		this._buttons = this._getButtons();
+		this._lastFocused = this._buttons.length ? this._buttons[0] : this._lastFocused;
+		this._setButtonsTabindex();
+	}
+
+	_setButtonsTabindex() {
+		this._buttons.forEach(b => {
+			b.tabindex = (b === this._lastFocused ? 0 : -1);
+		});
+	}
+
+	_handleClick(e) {
+		this._lastFocused = e.target;
+		this._setButtonsTabindex();
+	}
+
+	_handleKeyDown(e) {
```

Comment:
For the arrow keys, when the component is rendered right-to-left they need to reverse. Complicated I know! @dbatiste wrote something that might help -- [focusable arrow keys behavior](https://github.com/Brightspace/d2l-polymer-behaviors-ui).